config.go

```diff
@@ -90,6 +90,14 @@ type Config struct {
 	// If Pipeline is empty, no ingest pipeline will be specified in the Bulk request.
 	Pipeline string
 
+	// RequireDataStream forces the auto creation of the underlying index.
```

Comment:
This line doesn't read very right, although the explanation below is correct.

Reply:
you're right, just re-formatted it.